nmrfx-processor-gui/src/test/java/org/nmrfx/processor/gui/log/LogConsoleControllerTest.java

```diff
@@ -0,0 +1,159 @@
+package org.nmrfx.processor.gui.log;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.classic.spi.IThrowableProxy;
+import ch.qos.logback.classic.spi.LoggerContextVO;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.collections.transformation.FilteredList;
+import junit.framework.TestCase;
+import org.junit.Before;
+import org.junit.Test;
+import org.slf4j.Marker;
+
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.function.Predicate;
+
+public class LogConsoleControllerTest extends TestCase {
```

Comment:
extending TestCase is obsolete, and comes from a time before `@Test` annotations. It shouldn't be necessary.